ote/resources/public/language/fi.edn

```diff
@@ -512,7 +512,13 @@
  :dialog
  {:delete-transport-service
   {:title "Poista liikkumispalvelun tiedot?"
-   :confirm ["Haluatko varmasti poistaa \"" :name "\" liikkumispalvelun pysyvästi?"]}}
+   :confirm ["Haluatko varmasti poistaa \"" :name "\" liikkumispalvelun pysyvästi?"]}
+
+  :navigation-prompt
+  {:title "Sivulta poistumisen varmistus"
```

Comment:
Jotenki palikkasuomea.
Parempi otsikko, ois olla olematta ilman otsikkoa :)

En osaa nyt ehdottaa parempaa, kunhan kitisen.

Reply:
oisko Oululaisittain "Ookkonää ihan varma?"

Reply:
Tai jospa otsikkoon tiivistäis heti mistä virheessä on kyse?
:title "Lomakkeella on tallentamattomia tietoja"